pkg/authentication/basic/htpasswd.go

```diff
@@ -29,11 +29,16 @@ type sha1Pass string
 // NewHTPasswdValidator constructs an httpasswd based validator from the file
 // at the path given.
 func NewHTPasswdValidator(path string) (Validator, error) {
-	r, err := os.Open(path)
+	r, err := os.Open(path) // #nosec G304
```

Comment:
Might be good to have a comment explaining what we are preventing, here and elsewhere we are skipping gosec please